### DISCUSSION TITLE
Data Retention Settings

### DISCUSSION THREAD

**Discussion by tanto-satu** [featbit/featbit#656] [Q&A] [answered]
Hi featbit team,

I'm curious is there any data retention settings for audit logs, insight, users etc? for example i just want audit logs for only 3 months, the rest can be deleted for saving storage usage.

Thank you

**Comment by cosmic-flood** [comment, marked as answer]
Hi, there are no data retention settings for audit logs and insights. However, you can write a program to delete or archive audit logs that are older than 3 months.

**Reply by tanto-satu** [reply]
thanks for the answer!!

**Reply by thanh81** [reply]
hi - i'm interested in doing this. Can you point me in the direction of any exposed APIs to delete Insights/Events?  If not, any specific recommendations? It's self-hosted. If we were to go down the database direction, can you help isolate it to specific feature flags?
